### PR TITLE
Test and Completed Methods for highest and lowest scoring home and away teams

### LINKED ISSUE #28: #3: Highest scoring visitor method

### LINKED ISSUE #29: #3: Highest scoring home team method

### LINKED ISSUE #30: #3: Lowest scoring visitor method

### LINKED ISSUE #31: #3: Lowest scoring home team method

### PR DESCRIPTION
**Pull request by blake-enyart**
__X__ Wrote Tests
___X_ Implemented
____ Reviewed

# Neccesary checkmarks:
- [X] All Tests are Passing
- [X] The code will run locally

## Type of change
- [X] New feature
- [X] Bug Fix

# Implements/Fixes:
* description
closes #28 
closes #29 
closes #30 
closes #31 

# Check the correct boxes
- [X] This broke nothing
- [] This broke some stuff
- [] This broke everything

# Testing Changes
- [] No Tests have been changed
- [X] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [X] My code has no unused/commented out code
- [X] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [X] I have fully tested my code

# Please include a link to a gif of how you feel about this branch:
https://media.giphy.com/media/PuZ17QhVIoNFK/giphy.gif

### CODE REVIEW

File: test/team_test.rb [hunk 1]
```diff
@@ -4,16 +4,7 @@
...
-    team_path = './data/samples/team_info.csv'
-    game_teams_path = './data/samples/game_teams_stats.csv'
 
-    locations = {
-      games: game_path,
-      teams: team_path,
-      game_teams: game_teams_path
-    }
-    row = StatTracker.read_in_csv(locations[:teams]).readline
     @team = Team.new(team_id: "1", franchiseid:"23", shortname:"New Jersey", teamname:"Devils", abbreviation:"NJD", link:"/api/v1/teams/1")
```
Comment (erin-king) [inline test/team_test.rb:8]: Why are you hard coding line 17? This should be read from the file.

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -1,5 +1,3 @@
-require './lib/class_helper'
```
Comment (erin-king) [inline lib/stat_tracker.rb:1]: This should be required in stat_tracker.rb, not the runner. StatTracker directly needs to reference these things, not the runner.

**Review by erin-king — DISMISSED** [review]
See my comments on TeamTest setup and the removal for class_helper from StatTracker.

**Review by GlynnisOC — APPROVED** [review]
Looks great!

**Pull request merged**